README.md

```diff
@@ -1,108 +1,190 @@
 
-## Pre-requisites
+## 1. Pre-requisites
 
-To install OpenCTI using Docker, you will need the docker-compose command, you can install it using:
+`docker-compose`:
+### 🐧 Linux:
 
 ```bash
 $ sudo apt-get install docker-compose
 ```
+### ⌘ MacOS
+Download: https://www.docker.com/products/docker-desktop
+
+
 
 ## Clone the repository
 
 ```bash
-$ mkdir /path/to/your/app && cd /path/to/your/app
+$ mkdir -p /path/to/your/app && cd /path/to/your/app
 $ git clone https://github.com/OpenCTI-Platform/docker.git
 $ cd docker
 ```
 
-## Configure the environment
+## 3. Configure the environment
+
+Before running the `docker-compose` command, the `docker-compose.yml` file must be configured. 
 
-Before running the `docker-compose` command, the `docker-compose.yml` file must be configured.  Two ways to do that:
+There are two ways to do that:
 
-- Use environment variables as it is proposed and you have an exemple in the `.env.sample` file (ie. `APP__ADMIN__EMAIL=${OPENCTI_ADMIN_EMAIL}`).
-- Directly set the parameters in the `docker-compose.yml`.
+1. Use environment variables as it is proposed and you have an exemple in the `.env.sample` file (ie. `APP__ADMIN__EMAIL=${OPENCTI_ADMIN_EMAIL}`).
+1. Directly set the parameters in the `docker-compose.yml`.
 
- Whether you are using one method or the other, here are the mandatory parameters to fill:
+If setting within the environment, you can reference the methodology in the  [Environment setup on OpenCTI's Notion page](https://www.notion.so/Environment-setup-606996f36d904fcf8d434c6d0eae4a00#28b76731cae44cf0a59e70e4c84c795b
+) - located below for ease:
+
+### 🐧 Linux:
 
 ```bash
-OPENCTI_ADMIN_EMAIL=admin@opencti.io # Valid email address
-OPENCTI_ADMIN_PASSWORD=ChangeMe # String
-OPENCTI_ADMIN_TOKEN=ChangeMe # Valid UUIDv4
-MINIO_ROOT_USER=ChangeMeAccess # String
-MINIO_ROOT_PASSWORD=ChangeMeKey # String
-RABBITMQ_DEFAULT_USER=guest # String
-RABBITMQ_DEFAULT_PASS=guest # String
-CONNECTOR_HISTORY_ID=ChangeMe # Valid UUIDv4
-CONNECTOR_EXPORT_FILE_STIX_ID=ChangeMe # Valid UUIDv4
-CONNECTOR_EXPORT_FILE_CSV_ID=ChangeMe # Valid UUIDv4
-CONNECTOR_IMPORT_FILE_STIX_ID=ChangeMe # Valid UUIDv4
-CONNECTOR_IMPORT_FILE_PDF_OBSERVABLES_ID=ChangeMe # Valid UUIDv4
+sudo apt install -y jq
+
+cd ~/docker
+(cat <<EOF
+OPENCTI_ADMIN_EMAIL=admin@opencti.io
+OPENCTI_ADMIN_PASSWORD=CHANGEMEPLEASE
+OPENCTI_ADMIN_TOKEN=$(cat /proc/sys/kernel/random/uuid)
+MINIO_ROOT_USER=$(cat /proc/sys/kernel/random/uuid)
+MINIO_ROOT_PASSWORD=$(cat /proc/sys/kernel/random/uuid)
+RABBITMQ_DEFAULT_USER=guest
+RABBITMQ_DEFAULT_PASS=guest
+CONNECTOR_HISTORY_ID=$(cat /proc/sys/kernel/random/uuid)
+CONNECTOR_EXPORT_FILE_STIX_ID=$(cat /proc/sys/kernel/random/uuid)
+CONNECTOR_EXPORT_FILE_CSV_ID=$(cat /proc/sys/kernel/random/uuid)
+CONNECTOR_IMPORT_FILE_STIX_ID=$(cat /proc/sys/kernel/random/uuid)
+CONNECTOR_IMPORT_REPORT_ID=$(cat /proc/sys/kernel/random/uuid)
+DOCKER_IP=$(ip -j a show dev docker0 |jq -r '.[0].addr_info[0].local')
+EOF
+) > .env
```

Comment:
basically just ripped this from environment setup from the OpenCTI Notion site.

Reply:
Thanks for the inline commands, makes the init.py (https://github.com/OpenCTI-Platform/docker/blob/master/init.py) script kinda redundant.

Could you please remove the `DOCKER_IP` argument, since it is pointless in my eyes. I just noticed that the docs for the environment setup makes use of this variable, but I think there are better ways finding the backend services. I'll work on it...

Reply:
Yes, this should eliminate the need for the `init.py` script. I've added that in my PR.